inception-project/inception
# #4780 - Optionally include example documents in project templates

**What's in the PR**
- Include example documents with the named entity annotation project template

**How to test manually**
* Create a project using the "Entity annotation" template and check if documents are auto-imported when the switch is set and not if the switch is disabled.

**Automatic testing**

**Documentation**
